code/__DEFINES/loadout.dm

```diff
@@ -119,6 +119,7 @@ GLOBAL_LIST_INIT(medic_gear_listed_products, list(
 		/obj/item/clothing/glasses/hud/health = list(CAT_MEDSUP, "Medical HUD glasses", 2, "black"),
 		/obj/item/healthanalyzer/gloves = list(CAT_MEDSUP, "Health scanner gloves", 2, "black"),
 		/obj/item/defibrillator/gloves = list(CAT_MEDSUP, "Advanced medical gloves", 5, "black"),
+		/obj/effect/vendor_bundle/stretcher = list(CAT_ESS, "Medivac Stretcher", 45, "black"),
```

Comment:
Did you test this? This should probably be cat_medsup, not cat_ess.

Reply:
Changed, checked, tested, works.